src/Core/Style.js

```diff
@@ -755,14 +777,83 @@ class Style {
         return this;
     }
 
+    /**
+     * Applies the style.fill to a polygon of the texture canvas.
+     * @param {CanvasRenderingContext2D} txtrCtx The Context 2D of the texture canvas.
+     * @param {Path2D} polygon The current texture canvas polygon.
+     * @param {Number} invCtxScale The ratio to scale line width and radius circle.
+     * @param {Boolean} canBeFilled - true if feature.type == FEATURE_TYPES.POLYGON.
+     */
+    applyToCanvasPolygon(txtrCtx, polygon, invCtxScale, canBeFilled) {
+        // draw line or edge of polygon
+        if (this.stroke) {
+            // TO DO add posibility of using a pattern
```

Comment:
you can remove this todo since you created [#2210](https://github.com/iTowns/itowns/issues/2210) (or add the ticket number in the comment)

Reply:
done.
I did add the issue link in the code.